src/components/QuestionModal.jsx

```diff
@@ -17,13 +17,10 @@ const QuestionModal = props => {
     setShowingAnswer(!showingAnswer);
   };
 
-  const handleScoreCorrect = () => {
-    setScore({ ...score, [team]: score[team] + selected.value });
-    handleClose();
-  };
-
-  const handleScoreWrong = () => {
-    setScore({ ...score, [team]: score[team] - (selected.value/2) });
+  const handleScore = (correct = false) => {
+    correct === true ? 
+      setScore({ ...score, [team]: score[team] + selected.value }) : 
+      setScore({ ...score, [team]: score[team] - (selected.value/2) }); // if answered incorrectly team gets a penalty of half the possible points
```

Comment:
1. Ternary statements are hard to read. I would avoid them wherever an if/else statement will suffice. Also, there's no reason the answer should be incorrect by default. The chance of it being false/correct is virtually 50/50. 

2. Since we are not implementing Typescript yet, using conventions to imply our expected parameter type (boolean) is, for now, best IMO. So you can consider using the naming `isXXX` which usually implies a boolean (it either is, or is not).

Thus:

```
const handleScore = (isCorrectAnswer) => {
    if(isCorrectAnswer) setScore({ ...score, [team]: score[team] + selected.value }) : 
    else setScore({ ...score, [team]: score[team] - (selected.value/2) }); 
  };	
```

What do you think?

Reply:
Sure that makes sense. I'll send in an updated commit.